index.js

```diff
@@ -0,0 +1,25 @@
+const inquirer = require('inquirer')
```

Comment:
I've never used inquirer before but it looks like a very clever approach by taking the given schema and running it through your generate questions function.

Reply:
Definitely check out the package! It's very powerful.